test/controllers/books_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class BooksControllerTest < ActionDispatch::IntegrationTest
+  include FactoryGirl::Syntax::Methods
+
+  test 'should get index' do
+    get books_url
+    assert_response :success
+  end
```

Comment:
I'd be inclined to add a bit more into most of these tests, like at least do an assert_select to say that the title is displayed